### PR TITLE
Fix CI build of boost branch

### PR DESCRIPTION
**Pull request by fmauch**
The boost branch CI build got broken recently. This PR aims at fixing it.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
           tests/resources/dockerursim/build_and_run_docker_ursim.sh $URSIM_VERSION
         env: ${{matrix.env}}
       - name: install gtest
-        run: sudo apt-get install -y libgtest-dev
+        run: sudo apt-get install -y libgtest-dev libboost-all-dev
       - name: configure
         run: mkdir build && cd build && cmake .. -DBoost_INCLUDE_DIR=$BOOST_ROOT_1_72_0/include -DBUILDING_TESTS=1 -DINTEGRATION_TESTS=1
```
Comment (stefanscherzinger) [inline .github/workflows/ci.yml:28]: I'm not too sure about details here, but why the specific version of boost?
Is [our fix](https://github.com/UniversalRobots/Universal_Robots_Client_Library/pull/37/commits/c5de0327b99347b072a8d5fe60289111cb369f57) similar, or does that change something unexpected?

**Review by stefanscherzinger — APPROVED** [review]
Looks good.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (fmauch) [reply .github/workflows/ci.yml:28]: Yes, that should be checked, whether it can be removed.

Comment (fmauch) [reply .github/workflows/ci.yml]: Updated the PR accordingly.

**Pull request merged**